tests/zkevm/test_worst_compute.py

```diff
@@ -741,6 +741,105 @@ def test_worst_unop(
     )
 
 
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize("key_mut", [True, False])
+@pytest.mark.parametrize("val_mut", [True, False])
+def test_worst_tload(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    key_mut: bool,
+    val_mut: bool,
+):
+    """Test running a block with as many TLOAD calls as possible."""
+    env = Environment()
+
+    start_key = 41
+    start_val = 43
+    code_prefix = Op.PUSH1(start_val) + Op.PUSH1(start_key) + Op.JUMPDEST
+
+    # If `key_mut` is True, we mutate the key on every iteration of the big loop by mutiplying by
+    # a constant that should produce a long chain of distinct values.
+    code_key_mut = Op.PUSH1(3) + Op.MUL if key_mut else Bytecode()
```

Comment:
To mutate keys or values, I usually multiply by `3`. This is to have a long chain of different results mod 2**256. [Proof](https://math.stackexchange.com/a/2460162).